### DISCUSSION TITLE
Are labels required for all inputs?

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#824] [Q&A] [answered]
¿Does the input components in Paste need to have always a Label?

**Comment by TheSisb** [comment, marked as answer]
Hi @lgalloherran,

We strongly recommend adding a label to all inputs in Paste. This is for accessibility purposes and so that screen-reader software can determine what the input is asking.  More info can be found under the [accessibility section of our input component docs](https://paste.twilio.design/form-elements/input/#accessibility).  

Let us know if this helps address your question!
